src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeMortars.hpp

```diff
@@ -0,0 +1,179 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "Domain/CreateInitialMesh.hpp"
+#include "Domain/Direction.hpp"
+#include "Domain/Element.hpp"
+#include "Domain/Mesh.hpp"
+#include "Domain/Neighbors.hpp"
+#include "Domain/OrientationMap.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/Conservative/Tags.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/FluxCommunicationTypes.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/MortarHelpers.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Tags.hpp"
+#include "ParallelAlgorithms/Initialization/MergeIntoDataBox.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Frame {
+struct Inertial;
+}  // namespace Frame
+namespace Parallel {
+template <typename Metavariables>
+class ConstGlobalCache;
+}  // namespace Parallel
+/// \endcond
+
+namespace dg {
+namespace Actions {
+/*!
+ * \ingroup InitializationGroup
+ * \brief Initialize mortars between elements for exchanging fluxes
+ *
+ *  If the template parameter `AddFluxBoundaryConditionMortars`
+ *  is set to `false` then the mortar data for flux boundary conditions are not
+ *  initialized and other boundary conditions can be applied.
```

Comment:
Can you be more specific here?  Are the entries not created at all, or are they created in an unspecified state?

Reply:
Sure, but @nilsdeppe why did you actually add this option? I don't think it is used in any of the executables right now. Are you planning to use it? Else I can just remove the template parameter.

Reply:
Please don't remove it. It'll be used by generalized harmonic. The current way the BCs are imposed is horribly non-flexible and this alleviates that a little bit.

Reply:
ok, I added some more details to the documentation